packages/terra-application/src/application-base/ApplicationBase.jsx

```diff
@@ -40,26 +43,6 @@ const propTypes = {
    * If the `locale` prop is not provided, the preferred language from the browser will be used.
    */
   locale: PropTypes.string,
-  /**
-   * Custom translations for the current locale.
-   */
-  customTranslatedMessages: (props, propName, componentName) => {
```

Comment:
This is also a breaking change that should be noted in the changelog. We will no longer be supporting the ability for consumers to provide translations for an unsupported locale.